### ISSUE TITLE
Not render cyrillic character

### ISSUE THREAD

**Issue by KlimovDmitriy** [dompdf/dompdf#2236]
I used DomPDF to render HTML to PDF about a month. I used DejaVu Sans font and all was OK, but now text not render and show me blank document. If I change font to another, without utf-8 compatibility, document render, but show "?" instead normal character. 
What could be the problem? 
Here is my code:
```$html = '<html>
            <head>
                <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
                <style>
                    body { font-family: DejaVu Sans;padding-left: 30px;padding-right: 30px }, 
                    .row {
                        margin-left: -15px;
                        margin-right: -15px;
                    },
                    .row:before, .row:after {
                        content: " ";
                        display: table;
                    },
                    .logo{
                        width: 100%;
                    }
                    .row:after {
                        clear: both;
                    },
                    .col-md-4 {
                        width: 33.33333333333333%;
                    },
                    .col-md-6{
                        width: 50%;
                    },
                    .col-md-6,.col-md-4{
                        position: relative;
                        min-height: 1px;
                        padding-left: 15px;
                        padding-right: 15px;
                        float: left;
                    }
                </style>
            </head>
            <body>
                <!-- Some text -->
            </body>
        </html>';


$options = new Dompdf\Options();
$options->setDpi(300);
$dompdf = new Dompdf\Dompdf($options);
$html = mb_convert_encoding($html, 'HTML-ENTITIES', 'UTF-8');
$dompdf->loadHtml($html);
$dompdf->render();
$reference = 'Reference.pdf';
ob_end_clean();
$pdf = $dompdf->stream($reference, ['Attachment' => false]);
return $pdf;
exit;
```

**Comment by bsweeney** [comment]
I'm not having any issues. Perhaps it's the encoding conversion (`$html = mb_convert_encoding($html, 'HTML-ENTITIES', 'UTF-8');`) which should be unnecessary with more recent versions of Dompdf.

**Comment by KlimovDmitriy** [comment]
Problem fixed by rollback to 0.8.5 version
